forage-android/src/main/java/com/joinforage/forage/android/ForageSDKInterface.kt

```diff
@@ -0,0 +1,117 @@
+ * Use these methods in conjunction with the UI components ForagePANEditText
+ * and ForagePINEditText.
+ */
+internal interface ForageSDKInterface {
+
+    /**
+     * A method to securely tokenize an EBT card via ForagePANEditText
+     *
+     * @param params The parameters required for tokenization, including
+     * reference to a ForagePANEditText view for card input.
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the operation.
+     * On success, returns a [PaymentMethod](https://docs.joinforage.app/reference/create-payment-method)
+     * token which can be securely stored and used for subsequent transactions. On failure,
+     * returns a detailed error response for proper handling.
+     *
+     * @throws MissingForageConfigException If the passed ForagePANEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    suspend fun tokenizeEBTCard(params: TokenizeEBTCardParams): ForageApiResponse<String>
+
+    /**
+     * Checks the balance SNAP and EBT Cash balances of an EBT account via
+     * ForagePINEditText
+     *
+     * @param params The parameters required for tokenization, including
+     * reference to a ForagePINEditText and PaymentMethod ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the operation.
+     * On success, returns an object with `snap` and `cash` fields, whose values
+     * indicate the balance of each tender as of now
+     *
+     * @throws MissingForageConfigException If the passed ForagePANEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    suspend fun checkBalance(params: CheckBalanceParams): ForageApiResponse<String>
+
+    /**
+     * Captures a Forage Payment associated with an EBT card
+     *
+     * @param params The parameters required for payment capture, including
+     * reference to a ForagePINEditText and a Payment ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the
+     * payment capture. On success, returns a confirmation of the transaction.
+     * On failure, provides a detailed error response.
+     *
+     * @throws MissingForageConfigException If the passed ForagePANEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    suspend fun capturePayment(params: CapturePaymentParams): ForageApiResponse<String>
+}
+
+/**
+ * Data class representing the parameters required for tokenizing an EBT card.
+ *
+ * @property foragePANEditText A ForagePANEditText  UI component. Importantly,
+ * you must have called .setForageConfig() already
+ * @property customerId A unique identifier associated with a customer. This is
+ * required by FNS for fraud detection purposes
+ * @property reusable Indicates whether the tokenized card can be reused for
+ * multiple transactions.
+ */
+data class TokenizeEBTCardParams(
+    val foragePANEditText: ForagePANEditText,
```

Comment:
we use `foragePinEditText`,  but here we use `foragePANEditText`. 

What was the decision behind this? Can we use `foragePanEditText`?

Reply:
OMG yeah the casing stuff has been soo annoying. Sure let's do `foragePanEditText` and `foragePinEditText` for references to instances of ForagePANEditText and ForagePINEditText respectively. There's a pretty solid visual distinction which I think is a good thing here. I'll get on this now

Reply:
address in [f4e6a39](https://github.com/teamforage/forage-android-sdk/pull/97/commits/f4e6a39adade168d451899ec0d7f5e87fef066ca)